### PR TITLE
[MM-20226] Added aria-label to Invite People textbox

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
The new Invite People modal textbox was missing an `aria-label`. This PR adds the default `aria-label` to the email input based on its placeholder.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-20226

### CODE REVIEW

**Comment by asaadmahmood** [comment]
@devinbinnie Shouldn't we make it dynamic? So if users are added, then the aria-label may read out the users?
Or does it already do that? (Maybe it does because react-select may already have support for that).

**Comment by devinbinnie** [comment]
> @devinbinnie Shouldn't we make it dynamic? So if users are added, then the aria-label may read out the users?
> Or does it already do that? (Maybe it does because react-select may already have support for that).

I wanna say it already does that, I know with the `multiselect` component from the same library it will do that. But that's a separate concern to be handled elsewhere.

This is just to satisfy WCAG such that the input has an `aria-label` so that the input itself can be identified by a screen reader, rather than its contents.

File: components/widgets/inputs/users_emails_input.jsx [hunk 1]
```diff
@@ -278,6 +278,7 @@ class UsersEmailsInput extends React.Component {
                 onFocus={this.onFocus}
                 tabSelectsValue={true}
                 value={values}
+                aria-label={this.props.placeholder}
```
Comment (jespino) [inline components/widgets/inputs/users_emails_input.jsx]: This should be generic, and I don't think the `aria-label` should be bound to the `placeholder` property, I would request (as mandatory) a new `ariaLabel` property in the `UsersEmailsInput` (and in the `ChannelsInput` component), and then, from the outside I would use the `placeholder` if it makes sense.

**Pull request merged**